examples/react-datastore/src/App.tsx

```diff
@@ -9,7 +10,14 @@ function App() {
 
   const [mounted, setMounted] = useState<boolean>(false);
   const [addView, setAddView] = useState<boolean>(false);
-  const  { loading, error, data } = useFindTodos();
+  const  { isLoading: loading, error, data, subscribeToMore } = useFindTodos();
+  useEffect(() => {
```

Comment:
Why use effect is needed? Apollo doesn't have it

Reply:
See https://www.apollographql.com/docs/react/data/subscriptions/#subscribing-to-updates-for-a-query

Calling subscribeToMore in a `componentDidMount` is equivalent to calling it in a `useEffect`. The user can also provide a clean-up function to unsubscribe. In a class based component, you do the clean up in a `componentWillUnmount`